### PR TITLE
add django-log-outgoing-requests

### PR DESCRIPTION
**Pull request by Nath82**
#94

### CODE REVIEW

File: src/nrc/conf/includes/base.py [hunk 1]
```diff
@@ -303,6 +305,10 @@
     },
 }
 
+LOG_OUTGOING_REQUESTS_DB_SAVE = True
```
[the remark this answers is not in the document]
Comment (alextreme) [reply src/nrc/conf/includes/base.py:308]: In that case indeed set the default settings to False. I've opened #99 to pick up Sergei's additional request

**Comment by Nath82** [comment]
@alextreme it should be okay now

**Review by alextreme — APPROVED** [review]
This looks good to me. @annashamray feel free to double-check

**Review by annashamray — APPROVED** [review]
Looks good

**Pull request merged**